src/components/LandingComponent.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <body>
```

Comment:
Ich glaube den Body-Tag braucht man in Templates nicht. der wird automatisch von Vue eingefügt. Dann hätten wir am Ende zwei Body-Tags